pkg/metrics/exporter.go

```diff
@@ -119,3 +129,33 @@ func getMonitoredResource(projectID string) (*monitoredres.MonitoredResource, er
 		},
 	}, nil
 }
+
+// SetupMetrics initializes metrics reporting with the provided configuration
+func SetupMetrics(conf Config, server *httpserver.Server) (healthcheck.Handler, func()) {
```

Comment:
Can we make https://github.com/googleforgames/agones/blob/main/cmd/allocator/main.go#L221 to also use this method, and remove https://github.com/googleforgames/agones/blob/main/cmd/allocator/metrics.go#L35 if it's not used anywhere else?

Reply:
Okay, @gongmax. Yes, this (https://github.com/googleforgames/agones/blob/main/cmd/allocator/metrics.go#L35) is not used anywhere; I’ve checked it.